toolbox/fdc3-conformance/Intents-Tests.md

```diff
@@ -0,0 +1,240 @@
+| Test   | 3. IntentResolution   | The `raiseIntent` call returns an `IntentResolution` Object with an `AppIdentifer` as the `source field` with App E's `appId` and `instanceId` set.   |
+| Test   | 4. await results          | Test should `await resolution.getResult()` on the `IntentResolution` object returned in the previous step. A promise should be returned quickly.  |
+| E       | 5. return Channel | E should retrieve a Channel object via `fdc3.getOrCreateChannel("someChannelName")` and return it immediately. |
+| Test   | 6. receive Channel result | The promise received by Test from `resolution.getResult()` should resolve to a `Channel` object with the expected id. Confirm that the `type` of the Channel object is "app".
+| Test   | 7. addContextListener | Add a context listener to the Channel object via `channelObj.addContextListener("testContextZ", handler)` |
+| E       | 8. broadcast context | After a short delay (of a few seconds) E should broadcast a testContextZ context object over the channel, including an id field with a unique identifier set (e.g. a uuid). |
+| Test  | 9. receive context | Test should receive the context broadcast by E and confirm that it contains the expected id value. | 
+
+- `2.0-RaiseIntentChannelResult`: Perform the above test
+- `2.0-RaiseIntentPrivateChannelResult`: Perform the above test, but:
+  - Substitute app F through out - which returns a PrivateChannel result instead of channel.
+  - At step 5, the PrivateChannel should be created via`fdc3.createPrivateChannel()`.
+  - At step 6 confirm that the type of the channel is "private".
+
+### PrivateChannels cannot be accessed as app channels
+
+| App  | Step                | Details                                                                                                                                    |
+|-------|----------------|---------------------------------------------------------------------------------------------------|
+| Test   | 1. Create a private channel | Test creates a PrivateChannel with `const privChan = await fdc3.createPrivateChannel();`. Confirm that the Channel has an id.
+| Test   | 2. Confirm private channel id is unique | Test creates a second PrivateChannel with `const privChan = await fdc3.createPrivateChannel();`. Confirm that the Channel has an id and that it is distinct from the first channel created . |
+| Test   | 3. Retrieve as app channel | Attempt to retrieve the channels as App Channels with `const appChan = await fdc3.getOrCreateChannel(privChan.id)` this should fail with `ChannelError.AccessDenied`  |
+| Test   | 4. Raise Intent & await result | Start app J and pass it the id of the second PrivateChannel with `fdc3.raiseIntent("privateChanneliIsPrivate", privateChannelDetails)`, where the context object contains the id of the channel to attempt to retrieve. An IntentResolution should be returned and App J should start. Wait for a result to be returned via `await resolution.getResult()`.
```

Comment:
typo on `privateChanneliIsPrivate`, should be `privateChannelIsPrivate` (typo appears twice below)